resources/views/admin/registrations.blade.php

```diff
@@ -49,7 +49,7 @@
                                     @elseif($participant->latestPayment->paymentStatus == \App\Enums\PaymentStatus::failed)
                                         <span class="badge rounded-pill bg-danger">Gefaald</span>
                                     @elseif($participant->latestPayment->paymentStatus == \App\Enums\PaymentStatus::open)
-                                        <span class="badge rounded-pill bg-secondary">Open</span>
+                                        <span class="badge rounded-pill bg-secondary">Open</sp
```

Comment:
span niet goed afgesloten

```suggestion
                                        <span class="badge rounded-pill bg-secondary">Open</span>
```